core/dbt/parser/sources.py

```diff
@@ -154,7 +155,48 @@ def parse_source(self, target: UnpatchedSourceDefinition) -> SourceDefinition:
                 loaded_at_query = None
             else:
                 loaded_at_query = source.loaded_at_query
-        freshness = merge_freshness(source.freshness, table.freshness)
+
+        try:
+            project_freshness = FreshnessThreshold.from_dict(
+                self.root_project.sources.get("freshness", {})
+            )
+        except ValueError:
+            fire_event(
+                FreshnessConfigProblem(
+                    msg="Could not parse default freshness config in 'dbt_project.yml', ignoring.",
+                )
+            )
```

Comment:
Could we give the user slightly more information on why we couldn't parse the freshness config? I worry that without more context the user might not know what needs to change.